dspace/config/registries/openaire-cerif-types.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0"?>
+
+<dspace-dc-types>
+
+	<dspace-header>
+		<title>OpenAIRE Cris Types</title>
+	</dspace-header>
+
+	<dc-schema>
+		<name>oairecerif</name>
+		<namespace>https://www.openaire.eu/cerif-profile/1.1/</namespace>
+	</dc-schema>
+
+	<dc-type>
+		<schema>oairecerif</schema>
+		<element>author</element>
+		<qualifier>affiliation</qualifier>
+		<scope_note></scope_note>
```

Comment:
If we are going to keep this schema, then all these fields need Scope Notes.  We need to describe how these differ from similar fields in other schemas.